config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sidekiq.configure_client do |config|
+  config.redis = { size: 1 }
+end
+
+Sidekiq.configure_server do |config|
+  config.redis = { size: 7 }
+end
```

Comment:
it's correct, BUT
1. please, add `url` and `namespace` parameter to be more obvious
2. `url` should use `ENV` variable